### ISSUE TITLE
Number of epochs for pretrained

### ISSUE THREAD

**Issue by reddyanilkumar** [francoisgermain/SpeechDenoisingWithDeepFeatureLosses#4]
Could you please let me know the number of epochs the pre-trained model is trained.

**Comment by saurabh-kataria** [comment]
Since there are two models involved (speech enhancement network and the feature loss network), it would be good to know the number of epochs for both of them?

**Comment by francoisgermain** [comment]
The number of epochs for the training should correspond to the hard-coded one in the given scripts (i.e., 2500 epochs for loss network, 310 for speech enhancement network).
